crates/matrix-sdk-base/src/event_cache_store/traits.rs

```diff
@@ -25,7 +24,7 @@ use crate::media::MediaRequest;
 /// for the event cache of the SDK.
 #[cfg_attr(target_arch = "wasm32", async_trait(?Send))]
 #[cfg_attr(not(target_arch = "wasm32"), async_trait)]
-pub trait EventCacheStore: AsyncTraitDeps {
+pub trait EventCacheStore: Send + Sync {
```

Comment:
I suspect that this change will make it impossible to actually implement the trait on WASM (in `matrix-sdk-indexeddb`).

Reply:
Correct, I've put it back for now.